packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -800,6 +801,11 @@ mixin SchedulerBinding on BindingBase, ServicesBinding {
   /// Consider using [scheduleWarmUpFrame] instead if the goal is to update the
   /// rendering as soon as possible (e.g. at application startup).
   void scheduleForcedFrame() {
+    // TODO(chunhtai): Removes the if case once the issue is fixed
```

Comment:
This is the workaround, and it is the only change from the previous revert.
@xster